stripe.go

```diff
@@ -485,6 +370,107 @@ func (s *BackendConfiguration) ResponseToError(res *http.Response, resBody []byt
 	return stripeErr
 }
 
+// SetMaxNetworkRetries sets max number of retries on failed requests
+func (s *BackendConfiguration) SetMaxNetworkRetries(maxNetworkRetries int) {
+	s.MaxNetworkRetries = maxNetworkRetries
+}
+
+// Checks if an error is a problem that we should retry on. This includes both
+// socket errors that may represent an intermittent problem and some special
+// HTTP statuses.
+func (s *BackendConfiguration) shouldRetry(err error, resp *http.Response, numRetries int) bool {
```

Comment:
Minor but should it be `ShouldRetry`? (sorry I keep mixing those reviews with trying to learn from you, lmk if I should try to separate both)

Reply:
I think we want to keep it as `shouldRetry` so that it's not exported beyond the `stripe` package (it's just used a helper function by other functions on `BackendConfiguration`).

And just for clarity in case you haven't seen it before: whether a symbol is exported or not in Go depends on the case of its first letter. An uppercase letter means it's exported, and a lowercase letter mean it's not.

Reply:
oh I did not know that. Shouldn't it be in the private function list at the bottom in that case though?